app/services/calendar_service.rb

```diff
@@ -17,6 +17,14 @@ def weekend?(date)
     def weekend_or_holiday?(date)
       weekend?(date) || holiday?(date)
     end
+
+    def fiscal_start_date(time)
```

Comment:
thanks for adding this here! I'd like to make sure we clean up the other uses of this method, in `BaseReport` and such, if not in this PR let's add it in a follow-up

Reply:
Sure! i will create separate PR for this